providers/nodeReleasesProvider.tsx

```diff
@@ -1,12 +1,53 @@
 import { createContext } from 'react';
-import { useFetchNodeReleases } from '../hooks/useFetchNodeReleases';
+import { getNodeReleaseStatus } from '../util/nodeRelease';
+import NodeReleasesJSON from '../public/node-releases-data.json';
 import type { FC, PropsWithChildren } from 'react';
 import type { NodeRelease } from '../types';
 
+type NodeReleaseJSON = {
+  major: number;
+  version: string;
+  codename?: string;
+  currentStart: string;
+  ltsStart?: string;
+  maintenanceStart?: string;
+  endOfLife: string;
+  npm?: string;
+  v8?: string;
+  releaseDate?: string;
+  modules?: string;
+};
+
 export const NodeReleasesContext = createContext<NodeRelease[]>([]);
 
 export const NodeReleasesProvider: FC<PropsWithChildren> = ({ children }) => {
-  const releases = useFetchNodeReleases();
+  const data: NodeReleaseJSON[] = NodeReleasesJSON;
+  const now = new Date();
+
+  const releases = data.map(raw => {
```

Comment:
Memoization here would be nice. `useMemo(() => ...map(), [])`

The main intent here is that even tho the data is static, the .map is called on every re-render.